cumulus_etl/__init__.py

```diff
@@ -1,3 +1,3 @@
 """Cumulus public entry point"""
 
-__version__ = "1.1.0"
+__version__ = "1.1.1"
```

Comment:
Just a coincidence that this lined up like this, I didn't just do a search and replace :smile: